content/en/docs/concepts/configuration/poseidon-firmament-alternate-scheduler.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Poseidon-Firmament - An alternate scheduler
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+ 
+Poseidon is the [Firmament scheduler](https://github.com/Huawei-PaaS/firmament) integration for Kubernetes. At a very high level, Poseidon/Firmament scheduler augments the current Kubernetes scheduling capabilities by incorporating a new novel flow network graph based scheduling capabilities alongside the default Kubernetes Scheduler. It models the scheduling problem as a constraint-based optimization over a flow network graph – by reducing scheduling to a min-cost max-flow optimization problem.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+
+## Introduction	
+
+Poseidon is the [Firmament scheduler](https://github.com/Huawei-PaaS/firmament) integration for Kubernetes. At a very high level, Poseidon/Firmament scheduler augments the current Kubernetes scheduling capabilities by incorporating a new novel flow network graph based scheduling capabilities alongside the default Kubernetes Scheduler. It models the scheduling problem as a constraint-based optimization over a flow network graph – by reducing scheduling to a min-cost max-flow optimization problem.
+
```

Comment:
* Again, break down the descriptions of what this scheduler can do.
* In general, is this the best location for this page? There is information for configuring multiple schedulers under
tasks/administer-cluster/configure-multiple-schedulers.
* If this scheduler (Poseidon) and platform (Firmament) are in the alpha state, you should state that explicitly.

Reply:
- The language has been simplified and the sentences broken to shorter ones.  

- This document does not contain any steps that need to be performed. The current location was a tentative one. After reviewing the other pages we feel that the path (/concepts/extend-kubernetes) is a better fit. 
- @bsalamat Please review and advise.
- We have added the feature state tag at the top. The alpha status is also mentioned in the Current Project Stage.